src/styles/blocks/card__description.css

```diff
@@ -0,0 +1,15 @@
+.card__description {
+  position: absolute;
```

Comment:
Don't use `absolute` positioning for this task. It's not really necessary here. 
Learn to position elements without taking them out of the flow.
Use `margins` and `display: flex`